libraries/botbuilder/src/fileTranscriptStore.ts

```diff
@@ -13,16 +13,27 @@ import * as rimraf from 'rimraf';
 
 /**
  * The file transcript store stores transcripts in file system with each activity as a file.
+ *
+ * @remarks
+ * This class provides an interface to log all incoming and outgoing activities to the filesystem.
+ * It implements the features necessary to work alongside the TranscriptLoggerMiddleware plugin.
+ * When used in concert, your bot will automatically log all conversations.
+ *
+ * Below is the boilerplate code needed to use this in your app:
+ * ```javascript
+ * { FileTranscriptStore, TranscriptLoggerMiddleware } = require('botbuilder');
```

Comment:
Shouldn't this be "const { FileTranscriptStore, TranscriptLoggerMiddleware } = require('botbuilder');"